docs/build-windows.md

```diff
@@ -0,0 +1,35 @@
+Windows builds can be created using:
+
+- MSVC, using Visual Studio 2019 solution, and vcpkg for installing
+  dependencies (*recommended*).
+- Clang or GCC, using Meson buildsystem and MSYS2 environment for installing
+  dependencies..
```

Comment:
- Microsoft's MSVC compiler, Visual Studio 2019 IDE suite, and vcpkg to provide dependencies _(*recommended*)_.
- The Clang or GCC compilers using the Meson build system running within the MSYS2 environment to provide dependencies.

Reply:
"Microsoft's Microsoft Visual C compiler"? :) otherwise agree - edited.